analysis/report_templates/default.html

```diff
@@ -128,7 +128,7 @@ <h5 class="center-align">Mean coverage growth over time</h5>
             <div class="card-panel deep-orange lighten-3">
                 <div class="row valign-wrapper">
                     <div class="col s4 m2">
-                        <i class="medium material-icons">error</i>
+                        <center><i class="medium material-icons">error</i></center>
```

Comment:
does "center-align" inside class work, i think that is how is done in many places.

Reply:
Done